docs/quickstart.md

```diff
@@ -130,6 +130,8 @@ The following step will partition and format your disk, and mount it to `/mnt`.
 
 ```console
 sudo nix --experimental-features "nix-command flakes" run github:nix-community/disko -- --mode disko /tmp/disk-config.nix
+# or alternatively you can use nix-shell
```

Comment:
The difference here is that this comes from nixpkgs, which might be ok for the tutorial actually.